frontend/packages/dev-console/src/components/pipelines/PipelineDetails.tsx

```diff
@@ -12,28 +12,30 @@ const PipelineDetails = ({ obj: pipeline }) => (
       <div className="col-sm-6">
         <ResourceSummary resource={pipeline} />
       </div>
-      <div className="col-sm-6">
-        <SectionHeading text="Tasks" />
-        <dl>
-          {pipeline.spec.tasks.map((task) => {
-            return (
-              <React.Fragment key={task.name}>
-                <dt>Name: {task.name}</dt>
-                <dd>
-                  Ref:{' '}
-                  <ResourceLink
-                    kind={referenceForModel(TaskModel)}
-                    name={task.taskRef.name}
-                    namespace={pipeline.metadata.namespace}
-                    title={task.taskRef.name}
-                    inline
-                  />
-                </dd>
-              </React.Fragment>
-            );
-          })}
-        </dl>
-      </div>
+      {pipeline.spec.tasks && (
```

Comment:
`pipeline.spec` nor `pipeline.spec.tasks` are guaranteed by the type. Recommend handling the NPEs as well.

Also, could we get an explicit type on this variable - like type the FC?

Reply:
In `frontend/public/module/k8s/index.ts`, this type is used for other components that this component calls.
```
export type K8sResourceKind = {
  apiVersion?: string;
  kind?: string;
  metadata?: ObjectMetadata;
  spec?: {
    selector?: Selector;
    [key: string]: any
  };
  status?: {[key: string]: any};
  type?: {[key: string]: any};
  data?: {[key: string]: any};
};
```

Reply:
@andrewballantyne Done.